### ISSUE TITLE
Error/broken link "Penflip" of Markdown lesson

### ISSUE THREAD

**Issue by pao-chuan** [programminghistorian/jekyll#2150]
Hi, I'm Pao-Chuan, a new member that joining this program as the work placement of UCL. I'm working with Adam and feel really interesting and enjoyable, because I do learn something new continuously when checking if the tutorials/lessons still work properly.

When working on the lesson called "Sustainable Authorship in Plain Text using Pandoc and Markdown", I found that the link of "Penflip" appears an error after clicking on it in the "Useful Resources" section, I use Mac system to test this link through Google Chrome, Firefox and Safari, all the browsers announce “Your connection is not private." and it's not safe for user to visit anymore.

Besides, Adam and I  also tested this link through Window system, it doesn't work as well.

The screen shot of the error is provided.
![12171623318584_ pic](https://user-images.githubusercontent.com/85676300/121504438-ec088680-c9d9-11eb-8ce7-2656ca65e601.jpg)

**Comment by acrymble** [comment]
Thank you @pao-chuan for finding this problem. I've checked and this also affects the Spanish translation of the same lesson:

https://programminghistorian.org/es/lecciones/escritura-sostenible-usando-pandoc-y-markdown

The lesson is translated into French but this link is not listed in that version.

**Comment by acrymble** [comment]
@pao-chuan and I have looked at this @svmelton and @rivaquiroga and I am confident that this link can be removed from the lesson without any adverse effects to the learning. It is a suggested resource for further learning that seems no longer to be available and that presents a security warning when visited. I have suggested to @pao-chuan that she attempt to remove the link to fix this bug.

We will also update the "last tested" field for these two lessons.

We will make sure to ask your permission at the pull request stage.
